interceptors/Mementifier.cfc

```diff
@@ -200,30 +203,40 @@ component{
 		// Filter out exclude items and never include items
 		arguments.includes = arguments.includes.filter( function( item ){
 			return !arrayFindNoCase( excludes, item ) && !arrayFindNoCase( thisMemento.neverInclude, item );
-		} );
+        } );
 
 		// Process Includes
 		for( var item in arguments.includes ){
 
-			//writeDump( var="Processing: #item#" );abort;
+			// writeDump( var="Processing: #item#" );abort;
 
 			// Is this a nested include?
 			if( listLen( item,  "." ) > 1 ){
 				// Retrieve the relationship
 				item = listFirst( item, "." );
-			}
-
-			// Retrieve Value for transformation: ACF Incompats Suck on elvis operator
-			if( structKeyExists( this, "get#item#" ) ){
-				var thisValue = invoke( this, "get#item#" );
-				// Verify Nullness
-				thisValue = isNull( thisValue ) ? (
-					structKeyExists( thisMemento.defaults, item ) ? thisMemento.defaults[ item ] : $mementifierSettings.nullDefaultValue
-				) : thisValue;
-			} else {
-				// Calling for non-existent properties, skip out
-				continue;
-			}
+            }
+
+            // Retrieve Value for transformation: ACF Incompats Suck on elvis operator
+            var thisValue = javacast( "null", "" );
+
+            if ( arguments.trustedGetters || structKeyExists( this, "get#item#" ) ) {
+                try {
+                    thisValue = invoke( this, "get#item#" );
+                } catch ( any e ) {
+                    // Unless trusted getters is on and there is a mapper for this item rethrow the exception.
+                    if ( !arguments.trustedGetters || !structKeyExists( arguments.mappers, item ) ) {
```

Comment:
so what happens if trusted getter is on and there is an exception? Looks like it will be ignored? Why?

Reply:
If trusted getters is on and there is an exception it will rethrow the exception **UNLESS** there is also a mapper for that item, in which case it will treat it as null and move on.

It's not perfect, but I was trying to map it to Quick which throws an exception if the method is missing.  I could type hint Quick's exception (`QuickMissingMethod`) and only have this logic there?

Reply:
ok, but then the mapper is responsible to do null checks. I am fine with that. Can you make sure you add the docs for this in the readme please. Then we can merge